ui/raidboss/data/05-shb/ultimate/the_epic_of_alexander.js

```diff
@@ -3402,5 +3430,147 @@ let getHeadmarkerId = (data, matches) => {
+        '--alex untargetable--': '--알렉산더 타겟 불가능--',
+        'Aetheroplasm': '에테르 폭뢰',
+        'Almighty Judgment': '성스러운 대심판',
+        'Alpha Sword': '알파검',
+        'Apocalyptic Ray': '파멸 계시',
+        'Cascade': '폭포수',
+        'Chakrams': '차크람',
+        'Chastening Heat': '신벌의 열선',
+        'Collective Reprobation': '무리 단죄',
+        'Crashing Wave': '충격의 파도',
+        'Divine Judgment': '신성한 심판',
+        'Divine Spear': '신성한 불꽃',
+        'Double Rocket Punch': '양손 로켓 주먹',
+        'Down for the Count': '넉다운',
+        'Drainage': '하수로',
+        'Earth Missile': '대지 미사일',
+        'Embolus': '응고체',
+        'Enumeration': '계산',
+        'Eternal Darkness': '암흑의 운명',
+        'Exhaust': '고갈',
+        '/Stillness': '정지 명령',
+        'Fate: ': '미래: ',
+        'Fate Calibration': '미래 확정',
+        'Fate Projection': '미래 관측',
+        'Final Sentence': '사형 판결',
+        'Flarethrower': '대화염방사',
+        'Fluid Strike': '유체 강타',
+        'Fluid Swing': '유체 타격',
+        'Gavel': '최후의 심판: 폐정',
+        'Hand of Pain': '고통의 손길',
+        'Hand of Prayer': '기도의 손길',
+        'Hawk Blaster': '호크 블래스터',
+        'Hidden Minefield': '은폐 지뢰 살포',
+        'Inception(?! )': '시공 잠행',
+        'Inception Formation': '시공 잠행 대형',
+        'Incinerating Heat': '정화의 열선',
+        'Individual/': '개체/',
+        'Individual Reprobation': '개체 단죄',
+        'Irresistible Grace': '연대 형벌',
+        'J Jump': '정의의 점프',
+        'J Kick': '정의의 발차기',
+        'J Storm': '정의의 폭풍',
+        'Judgment Crystal': '심판의 결정체',
+        'Judgment Nisi': '임시처분',
+        'Limit Cut': '리미터 해제',
+        'Link-Up': '시스템 연결',
+        'Liquid Gaol': '액체 감옥',
+        'Mega Holy': '메가 홀리',
+        'Middle Blaster': '중앙 블래스터', // CHECKME
+        'Missile Command': '미사일 전탄 발사',
+        'Obloquy': '오명',
+        'Optical Sight': '조준',
+        'Ordained Capital Punishment': '가중 처벌',
+        'Ordained Motion': '행동 명령',
+        'Ordained Punishment': '처벌',
+        'Ordained Stillness': '정지 명령',
+        'Photon': '광자',
+        ' Players Remaining': '명 남음',
+        'Pressurize': '수압 충전',
+        'Propeller Wind': '추진 돌풍',
+        'Protean Wave': '변화의 물결',
+        'Punishing Wave': '응징의 파도',
+        'Radiant Sacrament': '원형 성례',
+        'Rage Wave': '분노한 액체 웨이브',
+        '(?<! )Repentance': '죄의식',
+        '(?<! )Sacrament': '십자 성례',
+        'Severity': '중벌', // CHECKME
+        'Sluice': '봇물',
+        'Solidarity': '연대', // CHECKME
+        'Spin Crusher': '회전 분쇄',
+        'Splash': '물장구',
+        'Summon Alexander': '알렉산더 소환',
+        'Super Blassty Charge': '슈퍼 블래스티 돌진',
+        'Super Jump': '슈퍼 점프',
+        'Surety': '서약', // CHECKME
+        'Temporal Interference': '시공 간섭',
+        'Temporal Prison': '시간의 감옥',
+        'Temporal Stasis': '시간 정지',
+        'The Final Word': '확정 판결',
+        'Throttles': '질식', // CHECKME
+        'Void Of Repentance': '참회의 방',
+        'Water and Thunder': 'Water and Thunder', // FIXME
```

Comment:
This is the water/thunder explosions.  Compressed Lightning and Compressed Water are the debuffs.  This entry is Crashing Thunder / Crashing Wave when the debuffs explode.